yb-voyager/cmd/analyzeSchema.go

```diff
@@ -921,6 +921,9 @@ sqlParsingLoop:
 			} else if matches := dollarQuoteRegex.FindStringSubmatch(currLine); matches != nil {
 				dollarQuoteFlag = 1 //denotes start of the code/body part
 				codeBlockDelimiter = matches[0]
+			} else if strings.Contains(currLine, "BEGIN ATOMIC") {
+				dollarQuoteFlag = 1 //denotes start of the sql body part https://www.postgresql.org/docs/15/sql-createfunction.html#:~:text=a%20new%20session.-,sql_body,-The%20body%20of
+				codeBlockDelimiter = "END"
```

Comment:
1. There are chances of `BEGIN ATOMIC` or `END` being on same line, different casing(smallcase, mixed case)
```
CREATE OR REPLACE FUNCTION subtract_and_log(a INT, b INT)
RETURNS INT
LANGUAGE SQL
BEGIN ATOMIC INSERT INTO test_log(action) VALUES ('Subtracting numbers'); RETURN a - b; END;
```

2. Using regexp here would be right way to go here, given there are N number of posibilities.

Reply:
> There are chances of BEGIN ATOMIC or END being on same line, different line, something on same line and rest on the next line.

This automatically works with the current code, added unit test case for that.

>Posibility of different casing(smallcase, mixed case) used for BEGIN ATOMIC ... END;

Fixed the casing by using the BEGIN ATOMIC with regexp but END with string lower upper both

Not trying to cover a lot cases given that we should use the pg-parser anyways but having this meanwhile to atleast able to detect the SQL body case in this PR #2165

Reply:
I doubt we are going to have pg-parser soon for breaking the sqlfile into sql statements.
Also for oracle cases where syntax is off, (not fully converted to PG), there the parser will fail, so we could either use current approach or  mix of both.

I would say lets cover most of it if we can, with small efforts.
Since this SQL body thing is a part of DDL, and our import schema can also break.